sormas-ui/src/main/java/de/symeda/sormas/ui/ActivityAsCase/ActivityAsCaseField.java

```diff
@@ -0,0 +1,241 @@
+ * *
+ * * This program is free software: you can redistribute it and/or modify
+ * * it under the terms of the GNU General Public License as published by
+ * * the Free Software Foundation, either version 3 of the License, or
+ * * (at your option) any later version.
+ * *
+ * * This program is distributed in the hope that it will be useful,
+ * * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * * GNU General Public License for more details.
+ * *
+ * * You should have received a copy of the GNU General Public License
+ * * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ * ******************************************************************************
+ */
+
+package de.symeda.sormas.ui.ActivityAsCase;
+
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.vaadin.ui.Window;
+import com.vaadin.v7.data.Property;
+import com.vaadin.v7.shared.ui.label.ContentMode;
+import com.vaadin.v7.ui.Label;
+import com.vaadin.v7.ui.Table;
+
+import de.symeda.sormas.api.EntityDto;
+import de.symeda.sormas.api.activityascase.ActivityAsCaseDto;
+import de.symeda.sormas.api.activityascase.ActivityAsCaseType;
+import de.symeda.sormas.api.caze.CaseDataDto;
+import de.symeda.sormas.api.contact.ContactDto;
+import de.symeda.sormas.api.event.TypeOfPlace;
+import de.symeda.sormas.api.exposure.ExposureDto;
+import de.symeda.sormas.api.i18n.Captions;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Strings;
+import de.symeda.sormas.api.user.UserDto;
+import de.symeda.sormas.api.user.UserRight;
+import de.symeda.sormas.api.utils.DataHelper;
+import de.symeda.sormas.api.utils.fieldaccess.UiFieldAccessCheckers;
+import de.symeda.sormas.api.utils.fieldvisibility.FieldVisibilityCheckers;
+import de.symeda.sormas.ui.UserProvider;
+import de.symeda.sormas.ui.caze.AbstractTableField;
+import de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent;
+import de.symeda.sormas.ui.utils.DateFormatHelper;
+import de.symeda.sormas.ui.utils.FieldAccessCellStyleGenerator;
+import de.symeda.sormas.ui.utils.VaadinUiUtil;
+
+@SuppressWarnings("serial")
+public class ActivityAsCaseField extends AbstractTableField<ActivityAsCaseDto> {
+
+	private static final String COLUMN_EXPOSURE_TYPE = ActivityAsCaseDto.ACTIVITY_AS_CASE_TYPE;
+	private static final String COLUMN_TYPE_OF_PLACE = ActivityAsCaseDto.TYPE_OF_PLACE;
+	private static final String COLUMN_DATE = Captions.date;
+	private static final String COLUMN_ADDRESS = Captions.address;
+	private static final String COLUMN_DESCRIPTION = ActivityAsCaseDto.DESCRIPTION;
+
+	private final FieldVisibilityCheckers fieldVisibilityCheckers;
+//	private Supplier<List<ContactReferenceDto>> getSourceContactsCallback;
+	private Class<? extends EntityDto> epiDataParentClass;
```

Comment:
Not necessary; ActivityAsCase will never be added to contacts, so the parent is always a case

Reply:
Removed

Reply:
Still seeing the field, are you sure it's gone?